spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns a successful response' do
+      get '/users/:user_id/posts'
+      expect(response).to be_successful
+    end
+
+    it 'renders the correct template' do
+      get '/users/:user_id/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users/:user_id/posts'
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+end
```

Comment:
- [ ] Please make sure to test for all controllers including the `Posts#show` controller. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.) requirement.